README.md

```diff
@@ -190,3 +190,5 @@ Get Bitcoin Testnet coins here: https://coinfaucet.eu/en/btc-testnet/
 Get Litecoin Testnet coins here: https://tltc.bitaps.com/
 
 Get Ethereum Goerli coins here: https://goerlifaucet.com/
+
+Get Ethereum Sepolia coins here: https://faucet.sepolia.dev/
```

Comment:
This site seems dead - are there other faucets?

Reply:
Fixed amount.test.tsx:147
but don't see what you mean for accounts_test.go:127